UnitTests/BTUIKAppearanceTests.m

```diff
@@ -1,13 +1,220 @@
 #import <XCTest/XCTest.h>
 
 #import "BTUIKAppearance.h"
+#import "UIColor+BTUIK.h"
+#import "BTDropInController.h"
 
 @interface BTUIKAppearanceTests : XCTestCase
 
 @end
 
 @implementation BTUIKAppearanceTests
 
+- (void)test_lightTheme_classMethod {
```

Comment:
We had a hard time coming up with good names for these tests to distinguish testing the deprecated class methods from the new `colorScheme` property. Open to suggestions!

Currently we have: 
- `test_lightTheme_classMethod` for the class method
- `test_lightTheme_enum` for the property
- `test_dynamicTheme_enum_usingSystemLightMode` for the dynamic color scheme property
- etc...